web/webpack.config.js

```diff
@@ -119,6 +119,12 @@ module.exports = function (env) {
               }
             }
           ]
+        }, 
+        {
+          test: /\.m?js/,
+          resolve: {
+              fullySpecified: false
+          }
```

Comment:
had to add this to get axios working
doesn't really make sense to me.

Reply:
axios must use .mjs files https://webpack.js.org/configuration/module/#resolvefullyspecified